---
title: "'Catch'는 'Try' 문 내에서 'Finally' 뒤에 사용할 수 없습니다."
ms.date: 07/20/2015
f1_keywords:
- vbc30379
- bc30379
helpviewer_keywords:
- BC30379
ms.assetid: 33d1278b-cf10-4c66-aaf8-08a4372f370b
ms.openlocfilehash: ab300d332dd82870901aec9bd57d8d5d8081a533
ms.sourcegitcommit: f8c270376ed905f6a8896ce0fe25b4f4b38ff498
ms.translationtype: MT
ms.contentlocale: ko-KR
ms.lasthandoff: 06/04/2020
ms.locfileid: "84403773"
---
# <a name="catch-cannot-appear-after-finally-within-a-try-statement"></a><span data-ttu-id="d287c-102">'Catch'는 'Try' 문 내에서 'Finally' 뒤에 사용할 수 없습니다.</span><span class="sxs-lookup"><span data-stu-id="d287c-102">'Catch' cannot appear after 'Finally' within a 'Try' statement</span></span>
<span data-ttu-id="d287c-103">`Catch` 문이 코드에서 `Finally` 문 블록을 끝내는 `Try` 뒤에 나타납니다.</span><span class="sxs-lookup"><span data-stu-id="d287c-103">A `Catch` statement appears in the code after the `Finally` ending a `Try` statement block.</span></span> <span data-ttu-id="d287c-104">`Catch` 는 `Try...Catch...Finally` 문 블록 내에서 사용해야 합니다.</span><span class="sxs-lookup"><span data-stu-id="d287c-104">`Catch` must appear within a `Try...Catch...Finally` statement block.</span></span>  
  
 <span data-ttu-id="d287c-105">**오류 ID:** BC30379</span><span class="sxs-lookup"><span data-stu-id="d287c-105">**Error ID:** BC30379</span></span>  
  
## <a name="to-correct-this-error"></a><span data-ttu-id="d287c-106">이 오류를 해결하려면</span><span class="sxs-lookup"><span data-stu-id="d287c-106">To correct this error</span></span>  
  
1. <span data-ttu-id="d287c-107">`Catch` 문을 코드의 더 적절한 위치로 이동합니다.</span><span class="sxs-lookup"><span data-stu-id="d287c-107">Move the `Catch` statement to a more appropriate place in the code.</span></span>  
  
## <a name="see-also"></a><span data-ttu-id="d287c-108">참고 항목</span><span class="sxs-lookup"><span data-stu-id="d287c-108">See also</span></span>

- [<span data-ttu-id="d287c-109">Try...Catch...Finally 명령문</span><span class="sxs-lookup"><span data-stu-id="d287c-109">Try...Catch...Finally Statement</span></span>](../language-reference/statements/try-catch-finally-statement.md)
